daemon/inertiad/webhook/webhook_test.go

```diff
@@ -24,13 +22,13 @@ func TestParse(t *testing.T) {
 		eventHeader string
 		eventValue  string
 	}{
-		{githubPushBody, "x-github-event", GithubPushHeader},
-		{gitlabPushBody, "x-gitlab-event", GitlabPushHeader},
-		{bitbucketPushBody, "x-event-key", BitbucketPushHeader},
+		{githubPushRawJSON, "x-github-event", GithubPushHeader},
+		{gitlabPushRawJSON, "x-gitlab-event", GitlabPushHeader},
+		{bitbucketPushRawJSON, "x-event-key", BitbucketPushHeader},
```

Comment:
💪

Still trying to convert the our tests to this format, didn't know about this standard until a month ago or so 😢